File: MODUL04/PraktukumC/5.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    scanf("%d %d", &n, &kelipatan);

    int i, j, total = 0;
    for (i = 1; i <= n; i++) {
        int sum = 0;
        for (j = i; j >= 1; j--) {
            sum += j * kelipatan;
            printf("(%d * %d) ", j, kelipatan);
        }
        total += sum;
        printf("= %d\n", sum);
    }

    printf("%d\n", total);

    return 0;
}

File: MODUL04/PraktikumPy/5.py
def main():
    n, kelipatan = map(int, input("").split())

    total = 0
    for i in range(1, n + 1):
        row_sum = 0
        for j in range(i, 0, -1):
            row_sum += j * kelipatan
            print(f"({j} * {kelipatan}) ", end="")
        total += row_sum
        print(f"= {row_sum}")

    print(total)

if __name__ == "__main__":
    main()
